src/sentry/integrations/github/client.py

```diff
@@ -750,13 +758,9 @@ def __init__(
         logging_context: Mapping[str, Any] | None = None,
     ) -> None:
         self.integration = integration
-        kwargs = {}
-        if hasattr(self.integration, "id"):
-            kwargs["integration_id"] = integration.id
-
         super().__init__(
             org_integration_id=org_integration_id,
             verify_ssl=verify_ssl,
             logging_context=logging_context,
-            **kwargs,
+            integration_id=getattr(self.integration, "id", None),
```

Comment:
this `getattr` shouldn't be necessary if the `integration: Integration` type is correct -- since that always has an `id: int`